.github/actions/templates/publishModule/scripts/Publish-ModuleToTemplateSpec.ps1

```diff
@@ -72,7 +72,7 @@ function Publish-ModuleToTemplateSpec {
         ##    EVALUATE RESOURCES   ##
         #############################
         if (-not (Get-AzResourceGroup -Name $componentTemplateSpecRGName -ErrorAction 'SilentlyContinue')) {
-            if ($PSCmdlet.ShouldProcess("Resource group [$componentTemplateSpecRGName] to location [$componentTemplateSpecRGLocation]", 'Deploy')) {
+            if ($PSCmdlet.ShouldProcess("Resource group [$RgmponentTemplateSpecRGName] to location [$componentTemplateSpecRGLocation]", 'Deploy')) {
```

Comment:
Not sure what this is. Seems a search and replace gone wrong?

Reply:
seems to be fixed